model/atmosphere/dycore/tests/dycore_stencil_tests/test_solve_tridiagonal_matrix_for_w_forward_sweep.py

```diff
@@ -54,20 +53,25 @@ def solve_tridiagonal_matrix_for_w_forward_sweep_numpy(
     return z_q, w
 
 
-def test_solve_tridiagonal_matrix_for_w_forward_sweep():
-    grid = SimpleGrid()
-    vwind_impl_wgt = random_field(grid, dims.CellDim, dtype=wpfloat)
-    theta_v_ic = random_field(grid, dims.CellDim, dims.KDim, dtype=wpfloat)
-    ddqz_z_half = random_field(grid, dims.CellDim, dims.KDim, dtype=vpfloat)
-    z_alpha = random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1}, dtype=vpfloat)
-    z_beta = random_field(grid, dims.CellDim, dims.KDim, dtype=vpfloat)
-    z_exner_expl = random_field(grid, dims.CellDim, dims.KDim, dtype=wpfloat)
-    z_w_expl = random_field(grid, dims.CellDim, dims.KDim, extend={dims.KDim: 1}, dtype=wpfloat)
+def test_solve_tridiagonal_matrix_for_w_forward_sweep(backend, grid):
+    if not helpers.is_gtfn(backend):
+        pytest.mark.xfail("Only supported by CPU backend")
```

Comment:
```suggestion
        pytest.mark.xfail("Only supported by GTFN backend")
```

Reply:
I was actuatlly wondering about that comment: whether it really neede CPU or is the GTFN backend...